### ISSUE TITLE
[BUG] UMI test profile stops at FASTQTOBAM

### ISSUE THREAD

**Issue by tijeco** [nf-core/sarek#611]
## Check Documentation

I have checked the following places for your error:

- [X] [nf-core website: troubleshooting](https://nf-co.re/usage/troubleshooting)
- [X] [nf-core/sarek pipeline documentation](https://nf-co.re/sarek/usage)

## Description of the bug

This in regards to the development version (3.0). I have discussed this with the author's in the sarek slack channel, but figured I would also put in an issue here. 

The umi test profile runs the NFCORE_SAREK:SAREK:CREATE_UMI_CONSENSUS workflow, but currently only runs `FASTQTOBAM` and finishes. It doesn't proceed to `BAM2FASTQ`, or `MAPPING_UMI`, `SAMBLASTER`, `GROUPREADSBYUMI` or `CALLUMICONSENSUS`.

## Steps to reproduce

This happens when running the following command with nextflow v21.10.6

```bash
nextflow run nf-core/sarek  -r dev -profile test,umi,docker
```
The commit hash used was `ffd1100`

This is the output

```
[e7/dcc056] process > NFCORE_SAREK:SAREK:PREPARE_GENOME:BWAMEM1_INDEX (genome.fasta)                       [100%] 1 of 1 ✔
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_GENOME:BWAMEM2_INDEX                                      -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_GENOME:DRAGMAP_HASHTABLE                                  -
[83/329894] process > NFCORE_SAREK:SAREK:PREPARE_GENOME:GATK4_CREATESEQUENCEDICTIONARY (genome.fasta)      [100%] 1 of 1 ✔
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_GENOME:MSISENSORPRO_SCAN                                  -
[2f/dcbac4] process > NFCORE_SAREK:SAREK:PREPARE_GENOME:SAMTOOLS_FAIDX (genome.fasta)                      [100%] 1 of 1 ✔
[8a/5a9418] process > NFCORE_SAREK:SAREK:PREPARE_GENOME:TABIX_DBSNP (dbsnp_146.hg38.vcf)                   [100%] 1 of 1 ✔
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_GENOME:TABIX_GERMLINE_RESOURCE                            -
[58/01dbca] process > NFCORE_SAREK:SAREK:PREPARE_GENOME:TABIX_KNOWN_INDELS (mills_and_1000G.indels.vcf)    [100%] 1 of 1 ✔
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_GENOME:TABIX_PON                                          -
[72/7f8f5f] process > NFCORE_SAREK:SAREK:PREPARE_INTERVALS:CREATE_INTERVALS_BED (genome.interval_list)     [100%] 1 of 1 ✔
[9a/59be0c] process > NFCORE_SAREK:SAREK:PREPARE_INTERVALS:GATK4_INTERVALLISTTOBED (genome)                [100%] 1 of 1 ✔
[05/8cc149] process > NFCORE_SAREK:SAREK:PREPARE_INTERVALS:TABIX_BGZIPTABIX_INTERVAL_SPLIT (chr22_1-40001) [100%] 1 of 1 ✔
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMENT_TO_FASTQ_INPUT:SAMTOOLS_VIEW_MAP_MAP                    -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMENT_TO_FASTQ_INPUT:SAMTOOLS_VIEW_UNMAP_UNMAP                -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMENT_TO_FASTQ_INPUT:SAMTOOLS_VIEW_UNMAP_MAP                  -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMENT_TO_FASTQ_INPUT:SAMTOOLS_VIEW_MAP_UNMAP                  -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMENT_TO_FASTQ_INPUT:SAMTOOLS_MERGE_UNMAP                     -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMENT_TO_FASTQ_INPUT:COLLATE_FASTQ_UNMAP                      -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMENT_TO_FASTQ_INPUT:COLLATE_FASTQ_MAP                        -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMENT_TO_FASTQ_INPUT:CAT_FASTQ                                -
[4c/441391] process > NFCORE_SAREK:SAREK:RUN_FASTQC:FASTQC (test-test_L1)                                  [100%] 1 of 1 ✔
[29/46486b] process > NFCORE_SAREK:SAREK:CREATE_UMI_CONSENSUS:FASTQTOBAM (test-test_L1)                    [100%] 1 of 1 ✔
[-        ] process > NFCORE_SAREK:SAREK:CREATE_UMI_CONSENSUS:BAM2FASTQ                                    -
[-        ] process > NFCORE_SAREK:SAREK:CREATE_UMI_CONSENSUS:MAPPING_UMI:BWAMEM1_MEM                      -
[-        ] process > NFCORE_SAREK:SAREK:CREATE_UMI_CONSENSUS:MAPPING_UMI:BWAMEM2_MEM                      -
[-        ] process > NFCORE_SAREK:SAREK:CREATE_UMI_CONSENSUS:MAPPING_UMI:DRAGMAP_ALIGN                    -
[-        ] process > NFCORE_SAREK:SAREK:CREATE_UMI_CONSENSUS:SAMBLASTER                                   -
[-        ] process > NFCORE_SAREK:SAREK:CREATE_UMI_CONSENSUS:GROUPREADSBYUMI                              -
[-        ] process > NFCORE_SAREK:SAREK:CREATE_UMI_CONSENSUS:CALLUMICONSENSUS                             -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMENT_TO_FASTQ_UMI:SAMTOOLS_VIEW_MAP_MAP                      -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMENT_TO_FASTQ_UMI:SAMTOOLS_VIEW_UNMAP_UNMAP                  -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMENT_TO_FASTQ_UMI:SAMTOOLS_VIEW_UNMAP_MAP                    -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMENT_TO_FASTQ_UMI:SAMTOOLS_VIEW_MAP_UNMAP                    -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMENT_TO_FASTQ_UMI:SAMTOOLS_MERGE_UNMAP                       -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMENT_TO_FASTQ_UMI:COLLATE_FASTQ_UNMAP                        -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMENT_TO_FASTQ_UMI:COLLATE_FASTQ_MAP                          -
[-        ] process > NFCORE_SAREK:SAREK:ALIGNMENT_TO_FASTQ_UMI:CAT_FASTQ                                  -
[-        ] process > NFCORE_SAREK:SAREK:GATK4_MAPPING:BWAMEM1_MEM                                         -
[-        ] process > NFCORE_SAREK:SAREK:GATK4_MAPPING:BWAMEM2_MEM                                         -
[-        ] process > NFCORE_SAREK:SAREK:GATK4_MAPPING:DRAGMAP_ALIGN                                       -
[-        ] process > NFCORE_SAREK:SAREK:MARKDUPLICATES:GATK4_MARKDUPLICATES                               -
[-        ] process > NFCORE_SAREK:SAREK:MARKDUPLICATES:BAM_TO_CRAM:SAMTOOLS_BAMTOCRAM                     -
[-        ] process > NFCORE_SAREK:SAREK:MARKDUPLICATES:BAM_TO_CRAM:SAMTOOLS_STATS_CRAM                    -
[-        ] process > NFCORE_SAREK:SAREK:MARKDUPLICATES:BAM_TO_CRAM:MOSDEPTH                               -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_RECALIBRATION:BASERECALIBRATOR                            -
[-        ] process > NFCORE_SAREK:SAREK:PREPARE_RECALIBRATION:GATHERBQSRREPORTS                           -
[-        ] process > NFCORE_SAREK:SAREK:RECALIBRATE:APPLYBQSR                                             -
[-        ] process > NFCORE_SAREK:SAREK:RECALIBRATE:MERGE_INDEX_CRAM:MERGE_CRAM                           -
[-        ] process > NFCORE_SAREK:SAREK:RECALIBRATE:MERGE_INDEX_CRAM:INDEX_CRAM                           -
[-        ] process > NFCORE_SAREK:SAREK:CRAM_QC:SAMTOOLS_STATS                                            -
[-        ] process > NFCORE_SAREK:SAREK:CRAM_QC:MOSDEPTH                                                  -
[-        ] process > NFCORE_SAREK:SAREK:SAMTOOLS_CRAMTOBAM_RECAL                                          -
[8e/d69cfd] process > NFCORE_SAREK:SAREK:CUSTOM_DUMPSOFTWAREVERSIONS (1)                                   [100%] 1 of 1 ✔
[e8/1eb686] process > NFCORE_SAREK:SAREK:MULTIQC                                                           [100%] 1 of 1 ✔
-[nf-core/sarek] Pipeline completed successfully-
```

## Log files

[dev_ffd1100.nextflow.log](https://github.com/nf-core/sarek/files/9012619/dev_ffd1100.nextflow.log)

**Comment by WackerO** [comment]
currently looking into this :)

**Comment by FriederikeHanssen** [comment]
closed by #614
